doc/users/doxygen.html

```diff
@@ -30,7 +30,7 @@
       Doxygen, if the online documentation is available,
       <a href="../doxygen/deal.II/index.html">here</a>. For older revisions,
 	links are provided here by revision number:
-	<a href="http://www.dealii.org/8.4.0/doxygen/deal.tag">8.4.0</a>,
+	<a href="http://www.dealii.org/8.4.1/doxygen/deal.tag">8.4.1</a>,
```

Comment:
By the way, we don't provide links to any of the other tag files of dot releases. Maybe not list 8.4.1 here?